spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -96,7 +98,7 @@
       expect(page).to have_content(I18n.t('doc_auth.errors.general.network_error'))
     end
 
-    it 'does not track state if state tracking is disabled' do
+    it 'does not track state if state tracking is disabled', allow_browser_log: true do
```

Comment:
```suggestion
    it 'does not track state if state tracking is disabled' do
```
is there a reason to add this? doesn't seem to be needed for this test